.github/workflows/express-build.yml

```diff
@@ -0,0 +1,110 @@
+# GitHub Action Workflow for building on Linux and Windows
+
+name: ExpressJS Express Build
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  
+  build:
+    name: Express Build
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: ["0.10", "0.12", "1.8", "2.5", "3.3", "4.9", "5.12", "6.17", "7.10", "8.17", "9.11", "10.19", "11.15", "12.16"]
+        os: [ubuntu-latest, windows-latest]
+        experimental: [false]
+        node-mirror: ["https://nodejs.org/dist/"]
+        include:
+          - node-version: 13
```

Comment:
We should just be testing on `13` as a normal release line.  Also, seems like we should add 14 and remove the nightly stuff for the normal tests.

Reply:
These switch overs in this repo have always been part of the minor release line changes. I would scope this actions pr to just changing what the current ci does over the actions and other changes like this can be separate, landing before or after this.

Reply:
Makes sense!  Similar to the `.npmrc` comment, just wanted to state it since it was here.

Reply:
Just to restate so I understand correctly:  you want to add 13 and 14 to the regular matrix, and remove the nightly build stuff?

Reply:
I've made this change

Reply:
Sorry, I missed your question. No, please do not update the tested versions in this PR.

Reply:
I'll get it fixed.